readme.md

```diff
@@ -39,11 +80,39 @@ Run:
 ## Documentation
 Refer to [documentation](docs/lioncore2mps-converter-design.adoc).
 
-## Publishing
+## Development process
+
+We have on "main" branch per supported MPS version, e.g. `mps2021.1`.
+We implement all new functionality on a branch, based on the oldest supported "main" branch.
+Example: The new branch `niko/great-new-feature` is based on `mps2021.1`.
+
+Once we merged the feature branch back to "main" (in the example: `mps2021.1`), we merge the changes into the next higher mps version branch. 
+Example:
+
+1. Develop on `niko/great-new-feature`, based on `mps2021.1`.
```

Comment:
Maybe mention explicitly you develop the feature on that branch using MPS 2021.1(.z)?